rholang/src/main/scala/coop/rchain/rholang/interpreter/util/RevAddress.scala

```diff
@@ -16,6 +16,9 @@ object RevAddress {
 
   private val tools = new AddressTools(prefix, keyLength = Validator.Length, checksumLength = 4)
 
+  def fromDeployerId(deployerId: Array[Byte]): Option[RevAddress] =
```

Comment:
Shouldn't we call this "fromPublicKey"?

Reply:
No, there's a `fromPublicKey` just next to it